recipes/django-material/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "django-material" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "5ad516f55f05e2fabb31797d4c5ae5819ae37c8945386d947f09eb550bf4463b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+
+test:
+  commands:
+    - python -c "import django; from django.conf import settings; settings.configure(INSTALLED_APPS=['material']); django.setup(); import material; import material.templatetags; import material.theme; import material.theme.amber; import material.theme.bluegrey; import material.theme.cyan; import material.theme.deeppurple; import material.theme.indigo; import material.theme.lightgreen; import material.theme.orange; import material.theme.purple; import material.theme.teal; import material.theme.blue; import material.theme.brown; import material.theme.deeporange; import material.theme.green; import material.theme.lightblue; import material.theme.lime; import material.theme.pink; import material.theme.red; import material.theme.yellow; import material.frontend; import material.frontend.management; import material.frontend.management.commands; import material.frontend.migrations; import material.frontend.templatetags; import material.frontend.views; import material.admin; import material.admin.templatetags;"
```

Comment:
I'm wondering if there is a better way to do this instead of a super long line. Maybe use a separate test script:

https://conda.io/docs/building/meta-yaml.html#run-test-script

Alternatively you could maybe put together a cleaner multiline block using a jinja template variable as in:

https://github.com/conda/conda-build/issues/644#issuecomment-155532255